packages/web-twig/src/Resources/components/Tabs/stories/TabsDefault.twig

```diff
@@ -1,9 +1,9 @@
 <TabList>
     <TabItem>
-        <TabLink isSelected id="pane1-tab" target="pane1">Item 1</TabLink>
+        <TabLink isSelected id="pane1-tab" data-toggle="tabs" target="pane1">Item 1</TabLink>
```

Comment:
This should be mentioned in the README as well. Currently, there is no indication that the user has to add this. There is no mention of the JS plugin at all, so maybe this whole section is missing. Please extend the docs.